### PR TITLE
Multisignature Transactions

### PR DESCRIPTION
**Pull request by nodar-chkuaselidze**
I haven't added this guide to `guide.html`. You can choose the place.

### CODE REVIEW

File: guides-markdown/multisig-tx.md [hunk 1]
```diff
@@ -0,0 +1,504 @@
...
+  // NOTE: that both wallets should be on the same index
+  // (depth) of derivation to geth the same addresses
+  // NOTE: Each time you createAddress index(depth) is
+  // incremented an new address is generated
+  const address1 = await wallet1.createAddress(wallet1account);
+  const address2 = await wallet2.createAddress(wallet2account);
+
+  // Address for both shouuld be the same
+  // Unless they were run separately. (Or by manually triggering API)
+  console.log(address1);
```
Comment (Bucko13) [inline guides-markdown/multisig-tx.md]: Maybe an assert to force these to be the same?

File: guides-markdown/multisig-tx.md [hunk 2]
```diff
@@ -0,0 +1,504 @@
...
+  console.log(address2);
+})().catch((e) => {
+  console.error(e);
+  process.exit(1);
+});
+```
+
+You will notice that we grab the `.account.accountKey`, first key is the xpubkey
+and both will be using xpubkey key derivation to come up with new addresses.
+You won't need to share any other public keys, they will derive them for you.
```
Comment (Bucko13) [inline guides-markdown/multisig-tx.md]: I feel like a link to the api documentation might be helpful here.

**Comment by Bucko13** [comment]
Looks great! My only last comment in addition to the small ones in the code is that it might be nice to have a short section at the end listing ways someone could build out on these examples.

File: guides-markdown/multisig-tx.md (hunk 1 above)
Comment (nodar-chkuaselidze) [reply guides-markdown/multisig-tx.md]: If someone is testing around, they might end up with different Addresses. (I did it too)
So assert will fail for sure if you have different depth in different accounts.

**Pull request merged**